packages/teams-js/test/public/calendar.spec.ts

```diff
@@ -301,14 +301,21 @@ describe('calendar', () => {
     });
   });
   describe('isSupported', () => {
-    it('should return false if the runtime says calendar is not supported', () => {
+    it('should return false if the runtime says calendar is not supported', async () => {
+      await utils.initializeWithContext(FrameContexts.content);
       utils.setRuntimeConfig({ apiVersion: 1, supports: {} });
       expect(calendar.isSupported()).not.toBeTruthy();
     });
 
-    it('should return true if the runtime says calendar is supported', () => {
+    it('should return true if the runtime says calendar is supported', async () => {
+      await utils.initializeWithContext(FrameContexts.content);
       utils.setRuntimeConfig({ apiVersion: 1, supports: { calendar: {} } });
       expect(calendar.isSupported()).toBeTruthy();
     });
+
+    it('should throw if called before initialization', () => {
+      utils.setRuntimeConfig(_uninitializedRuntime);
+      expect(() => calendar.isSupported()).toThrowError('The library has not yet been initialized');
```

Comment:
>'The library has not yet been initialized' [](http://example.com/codeflow?start=56&length=42)

Should this string be a constant that multiple places, including the product code, references?

Reply:
Done! Wow there are a lot